### PR TITLE
Remove LESS in favor of vanilla CSS via PostCSS/cssnext

### PR DESCRIPTION
**Pull request by koistya**
> PostCSS can do the same work as preprocessors like Sass, Less, and Stylus. But PostCSS is modular, 3-30 times faster, and much more powerful.

- [x] Convert `.less` files to `.css` (CSS4)
- [x] Configure [PostCSS](https://github.com/postcss/postcss) + [cssnext](http://cssnext.io/)
- [x] Remove `less`, `less-loader`

Ref #130, #126, #113

### CODE REVIEW

File: src/components/Footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  max-width: var(--max-content-width);
+  text-align: center;
+}
+
+.Footer-text {
+  color: rgba(255, 255, 255, .5);
+}
+
+.Footer-text--muted {
+  color: rgba(255, 255, 255, .3);
```
Comment (MoOx) [inline src/components/Footer/Footer.css]: `color(#fff a(.3))` can be more clear

File: webpack.config.js [hunk 2]
```diff
@@ -95,7 +90,11 @@ const config = {
     ]
   },
 
-  postcss: [autoprefixer(AUTOPREFIXER_BROWSERS)]
+  postcss: [
+    require('postcss-nested')(),
+    require('cssnext')(),
```
Comment (MoOx) [inline webpack.config.js]: you should use postcss-nested after cssnext (which handle the @import). Btw, cssnext have a "plugins" option to add you own plugin (the appropriate "cssnext" way)

File: webpack.config.js [hunk 3]
```diff
@@ -95,7 +90,11 @@ const config = {
...
+    require('autoprefixer-core')(AUTOPREFIXER_BROWSERS)
```
Comment (MoOx) [inline webpack.config.js]: cssnext is already included in autoprefixer (and cssnext also have a `browsers` option).

File: webpack.config.js [hunk 4]
```diff
@@ -65,10 +64,6 @@ const config = {
         loader: `${STYLE_LOADER}!${CSS_LOADER}!postcss-loader`
```
Comment (MoOx) [inline webpack.config.js]: you can use cssnext-loader + cssnext plugins options for nested. (also follow this https://github.com/cssnext/cssnext/issues/80)

**Comment by maxbates** [comment]
Tips for integrating Sass instead? I'm confused by the immediate adoption of PostCSS - it works great + fast, but very little IDE support.

**Comment by koistya** [comment]
@maxbates you would need to install `sass-loader`, then register to handle `.sass` files in `webpack.config.js`, then you can load them with `import './MyComponent.sass` or `import styles from './MyComponent'; @withStyles(styles) class MyComponent { render () { ... } }`

**Comment by koistya** [comment]
[Vote for PostCSS support in WebStorm](https://youtrack.jetbrains.com/issue/WEB-16601)

**Comment by maxbates** [comment]
Great, thanks a lot

File: webpack.config.js (hunk 3 above)
Comment (tobiasmuehl) [inline webpack.config.js]: You meant autoprefixer is included in cssnext I guess.

Comment (MoOx) [inline webpack.config.js]: yeah haha sorry :)

**Comment by iDVB** [comment]
Not sure if I missed something, but IS RSK currently using [cssnext](http://cssnext.io/features/)? Cause I see that it seems to be using alot of the same plugins.... but not directly using cssnext.

Is there a reason for that? Seems that cssnext was something that was originally looked into?

**Pull request merged**